packages/core/core/src/public/Environment.js

```diff
@@ -1,4 +1,5 @@
-// @flow
+// @flow strict-local
+// flowlint unsafe-getters-setters: off
```

Comment:
@devongovett @mischnic what do you think of allowing getters and setters globally? They do technically introduce potential unsoundness.

Reply:
👍  We do use them extensively in the public interfaces. And I think as long as we keep them side-effect-free (which no linter can verify anyway), it should be fine regarding soundness.